# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openh323/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- openh323_v1_18_0/include/ixjlid.h.vanilla	2007-04-20 19:04:44.261856344 +0200
+++ openh323_v1_18_0/include/ixjlid.h	2007-04-20 19:04:54.614446305 +0200
@@ -274,7 +274,6 @@
 
 #ifdef P_LINUX
 #include <linux/telephony.h>
-#include <linux/compiler.h>
 #include <linux/ixjuser.h>
 #endif
 
